proxy/src/main/java/org/apache/rocketmq/proxy/remoting/activity/ClientManagerActivity.java

```diff
@@ -82,6 +83,7 @@ protected RemotingCommand heartBeat(ChannelHandlerContext ctx, RemotingCommand r
                 this.remotingChannelManager.createProducerChannel(ctx.channel(), data.getGroupName(), clientId),
                 clientId, request.getLanguage(),
                 request.getVersion());
+            setClientPropertiesToChannelAttr(clientChannelInfo);
```

Comment:
Why do we need to add properties to the channel attribute when we have already passed the `clientId`, `language`, and `version` to `ClientChannelInfo`?

Reply:
Well, that's the problem. `ClientChannelInfo` only appears and can be accessed when client heartbeat. However, when it comes to message actions, which can not access `ClientChannelInfo`, their ProxyContext's  `clientId`, `language`, and `version` is empty. So in my pr, I put ClientChannelInfo to channel attr, in order that message actions' ProxyContext also have those client properties.